test/charts/trends/trends-chart.spec.js

```diff
@@ -55,4 +53,28 @@ describe('Directive: pfTrendsChart', function() {
     expect(angular.element(element).find('.trend-footer-pf').html()).toBe("Last 15 Minutes");
   });
 
+  it("should show the percentage in the trend heading", function() {
+
+    $scope.config.valueType = 'percentage';
+    $scope.$digest();
+
+    expect(angular.element(element).find('.trend-title-big-pf').html()).toBe("76%");
+    expect(angular.element(element).find('.trend-title-small-pf').html()).toBe("of 100 MHz");
+  });
+
+  it("should show large or small trend card layouts", function() {
+    // by default, should show a large card
+    var trendCard = angular.element(element.find('.trend-card-large-pf'));
+    expect(trendCard.size()).toBe(1);
+
+    $scope.config.layout = 'small';
+    $scope.$digest();
+    trendCard = angular.element(element.find('.trend-card-small-pf'));
+    expect(trendCard.size()).toBe(1);
```

Comment:
Is it worth ensuring that the opposite card isn't being shown here, perhaps by using [`hasClass()`](https://api.jquery.com/hasclass/)?